clients/py/README.md

```diff
@@ -15,7 +15,7 @@ from web3 import Web3
 from sapphirepy import sapphire
 
 # Setup your Web3 provider with a signing account.
-w3 = Web3(Web3.HTTPProvider('http://localhost:8545'))
+w3 = Web3(Web3.HTTPProvider(sapphire.NETWORKS.get('sapphire-localnet')))
```

Comment:
Let's fail here, if the network doesn't exist (e.g. typo...).

```suggestion
w3 = Web3(Web3.HTTPProvider(sapphire.NETWORKS['sapphire-localnet']))
```

Reply:
Agreed, changed dict access method.